docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+timesketch:
+    build: .
+    ports:
+        - "5000:5000"
+    links:
+        - elasticsearch
+        - postgres
+    environment:
+        - POSTGRES_USER=timesketch
+        - POSTGRES_PASSWORD=password
```

Comment:
Are these 'passwords' overwritten in the shell script later? Need a way to put random / pwgen values in here.